loki/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
General idea: for such long docs that won't be rendered on the website, consider adding a ToC at the beginning.